### ISSUE TITLE
Contacts with multiple phone numbers

### ISSUE THREAD

**Issue by rserentill** [JettF/JFContactsPicker#6]
Hello,
Thanks for this library!

It would be nice (in my case, necessary) if user could select a specific phone number for contacts with multiple phone numbers. User should be able to decide which number to select.

This could be achieved by listing multiple times a contact with multiple numbers. For example if user John Doe has numbers 123456 and 654321 it could show:

- John Doe (123456)
- John Doe (654321)

This is the simplest solution I could think of. A better one might be an independent selector asking the user which number he/she wants to select.

**Comment by JettF** [comment]
My apologies for taking so long to respond. I have been giving this some thought. I recognize the usefulness of this feature, but I am hesitant to display the same contact multiple times. As a user that would annoy me because iOS already handles a contact with multiple numbers so I am not expecting to see the same contact multiple times.

My next thought is to provide some sort of UI that pops up when a contact is selected that allows the user to select the phone number they want to use. Maybe an action sheet? This has its own nest of problems though because what if the user is interested in the contact's email instead? Now for each contact selected the user has to select a phone number and an email for each contact they want to use. From the POV of a user this seems overly slow.

In short, I agree that this library should include this functionality, but I am not sure how best to implement it right now.
